packages/mobile/src/tokens/saga.ts

```diff
@@ -248,3 +253,23 @@ export async function getCurrencyAddress(currency: Currency) {
       return contractKit.registry.addressFor(CeloContract.StableTokenEUR)
   }
 }
+
+export function* importTokenInfo() {
```

Comment:
It would also be nice if this function would has a few tests :D 

I think some test-cases to cover are:
- Happy path, having two tokens and seeing that their balances are fetched correctly.
- Have tokens with different number of decimals show up properly.
- If fetching one token fails, the rest are still returned and the failing one has the balance as `null`